packages/terra-table/src/_SelectableUtils.js

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
Remove the underline from the file if it's going to be exported with the SelectableTableRows component.